inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -515,7 +534,7 @@ int main(int argc, char* argv[]) {
                 slog::info << "Device: " << device << slog::endl;
                 for (const auto& cfg : supported_config_keys) {
                     try {
-                        slog::info << "  {" << cfg << " , " << exeNetwork.GetConfig(cfg).as<std::string>();
+                        slog::info << "  {" << cfg << " , " << ie.GetConfig(device, cfg).as<std::string>();
```

Comment:
As far as I remember here should be an executable network. @myshevts please comment

Reply:
yes, it should exec_network, as it turns the Hints (can be set to plugin but take an effect only when the network is known) into the real config (which is exactly printed here)